cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -332,6 +332,16 @@ data:
       </metric>
     </filter>
 
+    # This filter adds record label for google_cloud plugin. It will allow for future up-converting
+    # of resources during stackdriver resource type migration.
+    <filter **>
+      @type record_transformer
+      enable_ruby false
+      <record>
+        "logging.googleapis.com/compatibility" "true"
```

Comment:
This does not do what you think it does -- you want to add this label as part of the `<match ...>` directive with `@type google_cloud`.
That's already done as part of #59128.